apps/frontend/src/pages/register.tsx

```diff
@@ -48,8 +48,8 @@ const Register: React.FC = () => {
     return null;
   }
 
-  if (savedTap.tapResponse.chipIssuer === ChipIssuer.DEVCON_2024) {
-    return <RegisterDevcon savedTap={savedTap} />;
```

Comment:
Do we need this for testing purposes? I have some Devcon stickers and wound up using them yesterday to create a test account for Spotify OAuth App request. I may need to create other test accounts for other OAuth App requests.

Reply:
Yes, we should keep this around so we can give people the Devcon stickers in the meantime and have them test the flow